pkg/minikube/mustload/mustload.go

```diff
@@ -139,6 +139,23 @@ func Running(name string) ClusterController {
 	}
 }
 
+// HealthyOrNoKubernetes is a cmd-friendly way to load a healthy cluster
+// also allowing clusters without Kubernetes.
+func HealthyOrNoKubernetes(name string) ClusterController {
+	api, cc := Partial(name)
+
+	// if we're in a cluster that has Kubernetes deployed, expect it to be healthy.
+	if cc.KubernetesConfig.KubernetesVersion != constants.NoKubernetesVersion {
+		return Healthy(name)
+	}
+
+	// otherwise just add a node without Kubernetes.
+	return ClusterController{
```

Comment:
how would this affect output of "minikube profile list" ?

Reply:
This shows the output of 2 separate profiles:
```
$ minikube -p test1 start
$ minikube -p test1 start
$ minikube -p test2 node add
$ minikube -p test2 node add
$ minikube profile list     
|---------|-----------|---------|---------------|------|---------|---------|-------|--------|
| Profile | VM Driver | Runtime |      IP       | Port | Version | Status  | Nodes | Active |
|---------|-----------|---------|---------------|------|---------|---------|-------|--------|
| test1   | docker    | docker  | 192.168.121.2 | 8443 | N/A     | Stopped |     1 |        |
| test2   | docker    | docker  | 172.17.0.2    | 8443 | N/A     | Stopped |     3 |        |
|---------|-----------|---------|---------------|------|---------|---------|-------|--------|
```